frontend/public/components/RBAC/bindings.jsx

```diff
@@ -93,13 +97,22 @@ const menuActions = ({ subjectIndex, subjects }, startImpersonate) => {
     subjects.length === 1
       ? Kebab.factory.Delete
       : (kind, binding) => ({
-          label: i18next.t('bindings~Delete {{label}} subject', kind),
+          label: i18next.t('bindings~Delete {{kindLabel}} subject', {
+            kindLabel: kind.labelKey ? i18next.t(kind.labelKey) : kind.label,
```

Comment:
Probably better to have the logic that gets `kindLabel` once in this function to avoid duplication.